src/wallet/api/wallet_manager.cpp

```diff
@@ -81,6 +81,12 @@ bool WalletManagerImpl::closeWallet(Wallet *wallet)
 
 bool WalletManagerImpl::walletExists(const std::string &path)
 {
+    bool keys_file_exists;
+    bool wallet_file_exists;
+    tools::wallet2::wallet_exists(path, keys_file_exists, wallet_file_exists);
+    if(keys_file_exists){
+        return true;
+    }
```

Comment:
Right.  You wan't me to rebase with squash or just add a commit? @moneromooo-monero